conftest.py

```diff
@@ -13,6 +13,9 @@ def pytest_generate_tests(metafunc):
 collect_ignore = [
     "setup.py",
     "speechbrain/lobes/models/huggingface_transformers/mert.py",
+    "speechbrain/lobes/models/flair",
```

Comment:
Why are we adding these now?

Reply:
Flair is not compatible with numpy 2.0 and can't be run on the CI server anymore. The others started failing too for some reason.